src/NServiceBus.AcceptanceTests/Recoverability/Retries/When_custom_policy_always_moves_to_error.cs

```diff
@@ -37,15 +37,11 @@ public RetryEndpoint()
                 {
                     var scenarioContext = (Context) context.ScenarioContext;
                     configure.EnableFeature<TimeoutManager>();
-                    configure.SecondLevelRetries().CustomRetryPolicy(RetryPolicy);
+                    configure.Recoverability()
+                        .CustomPolicy((cfg, errorContext) => RecoverabilityAction.MoveToError());
                     configure.Notifications.Errors.MessageSentToErrorQueue += (sender, message) => { scenarioContext.MessageSentToErrorQueue = true; };
                 })
-                    .WithConfig<SecondLevelRetriesConfig>(c => c.TimeIncrease = TimeSpan.FromMilliseconds(1));
-            }
-
-            static TimeSpan RetryPolicy(SecondLevelRetryContext slrRetryContext)
-            {
-                return TimeSpan.MinValue;
+                .WithConfig<SecondLevelRetriesConfig>(c => c.TimeIncrease = TimeSpan.FromMilliseconds(1));
```

Comment:
Ok

On Monday, July 18, 2016, Tomasz Masternak <notifications@github.com> wrote:

> In
> src/NServiceBus.AcceptanceTests/Recoverability/Retries/When_custom_policy_always_moves_to_error.cs
> <https://github.com/Particular/NServiceBus/pull/3939#discussion_r71109312>
> :
>
> >                      configure.Notifications.Errors.MessageSentToErrorQueue += (sender, message) => { scenarioContext.MessageSentToErrorQueue = true; };
> >                  })
> > -                    .WithConfig<SecondLevelRetriesConfig>(c => c.TimeIncrease = TimeSpan.FromMilliseconds(1));
> > -            }
> > -
> > -            static TimeSpan RetryPolicy(SecondLevelRetryContext slrRetryContext)
> > -            {
> > -                return TimeSpan.MinValue;
> > +                .WithConfig<SecondLevelRetriesConfig>(c => c.TimeIncrease = TimeSpan.FromMilliseconds(1));
>
> I would rather remove it. Otherwise we get a test that has unclear focus
> and we test passing the configuration with When_custom_policy_provided
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Particular/NServiceBus/pull/3939/files/01ebaf4544e008dd1bfbc4f77619f32501dffa04#r71109312>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAKosjhV-zvBXJgSox-QJiTCQ5svl356ks5qWy-0gaJpZM4JMhh_>
> .
>